recipes/blaze/all/conanfile.py

```diff
@@ -0,0 +1,20 @@
+import os
+from conans import ConanFile, tools
+
+
+class BlazeConan(ConanFile):
+    name = "blaze"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://bitbucket.org/blaze-lib/blaze"
+    description = "open-source, high-performance C++ math library for dense and sparse arithmetic"
+    topics = ("conan", "blaze", "math", "algebra", "linear algebra", "high-performance")
+    license = "New (Revised) BSD license"
+
+    no_copy_source = True
+
```

Comment:
```suggestion

    @property
    def _source_subfolder(self):
        return "source_subfolder"

```

Reply:
@igl42 You can remark my comments as resolved. Then these won't show up in review anymore.